app/services/articles/feed.rb

```
@@ -7,7 +7,7 @@ def initialize(number_of_articles: 35, page: 1, tag: nil)
     end
 
     def published_articles_by_tag
-      articles = Article.published.limited_column_select.page(@page).per(@num_articles)
+      articles = Article.published.limited_column_select.page(@page).per(@number_of_articles)
```

I noticed that this was silently using the wrong var.